src/main/java/mate/academy/internetshop/dao/hibernate/BucketDaoHibernateImpl.java

```diff
@@ -0,0 +1,85 @@
+package mate.academy.internetshop.dao.hibernate;
+
+import java.util.List;
+import mate.academy.internetshop.dao.BucketDao;
+import mate.academy.internetshop.lib.annotation.Dao;
+import mate.academy.internetshop.model.Bucket;
+import mate.academy.internetshop.model.Item;
+import mate.academy.internetshop.util.HibernateUtil;
+import org.apache.log4j.Logger;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+@Dao
+public class BucketDaoHibernateImpl implements BucketDao {
+    private static Logger logger = Logger.getLogger(BucketDaoHibernateImpl.class);
+
+    @Override
+    public Bucket create(Bucket bucket) {
+        Transaction transaction = null;
+        Long bucketId = null;
+        try (Session session = HibernateUtil.sessionFactory().openSession()) {
+            transaction = session.beginTransaction();
+            bucketId = (Long) session.save(bucket);
+            transaction.commit();
+        } catch (Exception e) {
+            logger.error("Can't create bucket", e);
+            if (transaction != null) {
+                transaction.rollback();
+            }
+        }
```

Comment:
@kozhukhovsky add finally block and close session

Reply:
fixed all in next PR
https://github.com/kozhukhovsky/jv-internet-shop/pull/12